command.go

```diff
@@ -3840,3 +3840,54 @@ func (cmd *ZSliceWithKeyCmd) readReply(rd *proto.Reader) (err error) {
 
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+type LCSMatch struct {
+	Positions1 []int
+	Positions2 []int
+}
+
+type LCSMatchLen struct {
+	Match LCSMatch
+	Len int64
+}
+
```

Comment:
@monkey92t @vmihailenco  How does this look like for the command response for LCS with Len and Can you suggest a good structure for MINMATCHLEN and  WITHMATCHLEN?
Should I try to create a common structure? Any suggestions for that